src/site/components/navigation-sidebar-trigger.html

```diff
@@ -8,37 +8,104 @@
 used in a template with navigation-sidebar.html
 
 {% endcomment %}
-<button type="button" class="va-btn-sidebarnav-trigger vads-u-width--full"
-  aria-controls="va-detailpage-sidebar" id="sidebar-nav-trigger">
-  <span>
-    <b>More in this section</b>
-    {% include "src/site/assets/img/arrow-right-white.svg" %}
-  </span>
-</button>
-
+<div class="va-btn-sidebarnav-trigger" id="sidebar-nav-trigger">
```

Comment:
Does the movement of this class and ID change which element the click event is bound to? Just want to make sure we're not binding a click to a div and it's still on the button element to keep this keyboard accessible. https://www.w3.org/WAI/WCAG21/quickref/#keyboard

Reply:
no the `button` is still the thing that's clickable and using the keyboard to interact with this still works great 👍  This is the corresponding change https://github.com/department-of-veterans-affairs/veteran-facing-services-tools/pull/581/files#diff-058155fb4dee49e02a161d9d2c6f3e45d3e8dc90c87a77084f9eb5ab35178bdbR153-R155

Reply:
wonderful, thank you